saleor/static/scss/components/_header.scss

```diff
@@ -39,19 +39,21 @@
       width: auto;
     }
   }
-  @media (max-width: 370px) { 
+  @media (max-width: 370px) {
     svg {
       max-height: 25px;
       width: auto;
     }
   }
   &__login {
+    width: 100%;
```

Comment:
Why do we suddenly have to set width on so many selectors?

Reply:
Beacuse bootstrap's .container class has fixed `max-width` and `width: 100%` since beta. In alpha .container had fixed `width` and `max-width: 100%`.

Reply:
I'm confused. So you're saying that it already has the correct width?

Reply:
I'm saying that we need to set `width` on the parent element since .container is now scaling with the parent, not the other way like it used to be before.